arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -702,6 +702,33 @@ def rule_gfm_1_3_20(
         )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_3_21(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.3.21] Remove the tuple definition.
+    """
+    tupleConcepts = set()
+    concepts = pluginData.getExtensionConcepts(val.modelXbrl)
+    for concept in concepts:
+        if concept.namespaceURI == XbrlConst.xsd:
+            if concept.isTuple == True:
```

Comment:
Use the boolean value directly instead of comparing to True.
```suggestion
            if concept.isTuple:
```

Or more succinctly: 
```python
tupleConcepts = [
    concept for concept in pluginData.getExtensionConcepts(val.modelXbrl)
    if concept.isTuple
]
```